src/renderers/index.ts

```diff
@@ -31,7 +32,8 @@ export
 class HTMLWidget extends Widget {
   constructor(html: string) {
     super();
-    this.node.innerHTML = html;
+    let range = document.createRange();
+    this.node.appendChild(range.createContextualFragment(html));
```

Comment:
But that fix doesn't seem to have yet made its way into existing Safari implementations. Rendermime would be broken on not only current desktop impl, but all iOS devices as well.